desafio10.sql

```diff
@@ -0,0 +1,13 @@
+CREATE FUNCTION quantidade_musicas_no_historico(user_code INT)
+RETURNS INT READS SQL DATA
+BEGIN
+  DECLARE quantidade INT;
+  SELECT
+    COUNT(*)
+  FROM SpotifyClone.historico_reproducoes AS hr
+  INNER JOIN SpotifyClone.usuario AS usr
+  ON hr.usuario_id = usr.usuario_id
+  WHERE usr.usuario_id = user_code
```

Comment:
```suggestion
  INNER JOIN SpotifyClone.usuario AS usr
  ON hr.usuario_id = usr.usuario_id AND usr.usuario_id = user_code
```

Aqui você conseguiria não usar `WHERE` pq se usar `AND`, o WHERE não fica tão necessário hehehe